sdk/core/ci.yml

```diff
@@ -44,6 +44,8 @@ extends:
     Artifacts:
     - name: Azure.Core
       safeName: AzureCore
+      triggeringPaths:
+      - /sdk/resourcemanager/
```

Comment:
@m-nash do you want changes under the resourcemanager to trigger the core package validation? Or is that too much now?